mk/testcases/debian-10.6

```diff
@@ -0,0 +1 @@
+10.6
```

Comment:
why Debian doesn't have a vendor? Just out of interest.

Reply:
That's is all the `/etc/debian_version` file contains.